app/src/main/java/com/devshish/internship/data/repository/TokenRepository.kt

```diff
@@ -0,0 +1,6 @@
+package com.devshish.internship.data.repository
+
+class TokenRepository {
+
+    var token: String? = null
+}
```

Comment:
1. I suggest create Interface + Implementation
like this :

```
interface TokenRepository {
    val token: String?
    suspend fun clear()
}
```

Implementation should use SharedPreferences (to store token all time , not while app is working)